docs/docs/application-basics.md

```diff
@@ -21,19 +22,19 @@ You would use the following application definition (in JSON format) to describe
 }
 ```
 
-Note that `cmd` in the above example is the command that gets executed. Its value is wrapped by the underlying Mesos executor via `/bin/sh -c ${cmd}`.
+Note that `cmd` in the above example is the command that is executed. Its value is wrapped by the underlying Mesos executor via `/bin/sh -c ${cmd}`. <!-- this last sentence: why is it relevant to the user at this point? -->
```

Comment:
Because you can rely on the shell behavior like e.g. environment variable expansion.